test_scripts/Polices/ATF_Preloaded_PT_validation.lua

```diff
@@ -53,6 +53,25 @@ local function userPrint( color, message)
   print ("\27[" .. tostring(color) .. "m " .. tostring(message) .. " \27[0m")
 end
 
+local function TryStartSDL(self)
```

Comment:
@VVeremjova please extract from this funciton waiting for stop SDL and calling  SDL and use them in postconditions for function before start new SDL. Also please create defecct that SDL sometimes does not stops